charts/identity-server/values.prod.yaml

```diff
@@ -280,7 +280,7 @@ services-auth-delegation-api:
     XROAD_NATIONAL_REGISTRY_SERVICE_PATH: 'IS/GOV/6503760649/SKRA-Protected/Einstaklingar-v1'
     XROAD_RSK_PROCURING_ACTOR_TOKEN: 'true'
     XROAD_RSK_PROCURING_PATH: 'IS/GOV/5402696029/Skatturinn/relationships-v1'
-    XROAD_RSK_PROCURING_REDIS_NODES: '["clustercfg.general-redis-cluster-group.dnugi2.euw1.cache.amazonaws.com:6379"]'
+    XROAD_RSK_PROCURING_REDIS_NODES: '["clustercfg.general-redis-cluster-group.whakos.euw1.cache.amazonaws.com:6379"]'
```

Comment:
services-auth-* shouldn't change as it is deployed in the identity server namespace.

Reply:
fixed